server/db/025-add-notifications.sql

```diff
@@ -0,0 +1,42 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 25;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        CREATE TABLE IF NOT EXISTS notifications (
+            id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+            location_id text REFERENCES locations (locationid) NOT NULL,
```

Comment:
Interesting. I think you must have started this work before the `clients` table existed. I think this makes more sense for notifications to be at the `client` level than the `location`.